solr/core/src/java/org/apache/solr/filestore/FileStore.java

```diff
@@ -38,8 +42,13 @@ public interface FileStore {
    */
   void put(FileEntry fileEntry) throws IOException;
 
-  /** read file content from a given path */
-  void get(String path, Consumer<FileEntry> filecontent, boolean getMissing) throws IOException;
+  /**
+   * Read file content from a given path.
+   *
+   * <p>TODO: Is fetchMissing actually used? I don't see it being used, but the IDE doesn't flag it
+   * not being used!
```

Comment:
Lucene/Solr devs use a "nocommit" comment for something like this.  Precommit will fail so we remember to address it.  I also configure IntelliJ's "TODO" feature to consider "nocommit" and thus it shows in blood red (gets my attention)